proca/lib/proca_web/resolvers/action.ex

```diff
@@ -172,7 +172,20 @@ defmodule ProcaWeb.Resolvers.Action do
            get_action_page(params)
          end)
          |> Multi.run(:supporter, fn _repo, %{action_page: action_page} ->
-           get_supporter(action_page, params)
+           case get_supporter(action_page, params) do
+             {:ok, supporter = %Supporter{}} ->
+               {:ok, supporter}
+
+             {:ok, cref} ->
+               if Map.has_key?(params.action, :mtt) do
```

Comment:
I think we can also just check if action_type == "mtt"
@tttp What do you think?